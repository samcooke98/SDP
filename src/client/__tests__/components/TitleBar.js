import React from "react";
import renderer from "react-test-renderer";
// import ReactDOM from 'react-dom';

import TitleBar from "../../components/TitleBar/Titlebar";


describe('React Component: TitleBar', () => {
    it("matches a snapshot", () => {
        const tree = renderer.create(
            <TitleBar />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    })
})