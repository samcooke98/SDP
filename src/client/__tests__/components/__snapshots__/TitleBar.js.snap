// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React Component: TitleBar matches a snapshot 1`] = `
<div
  className={undefined}
>
  <h1>
    Scriptum
  </h1>
</div>
`;
